### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by bvanessen**

### CODE REVIEW

File: docs/publications.rst [hunk 1]
```diff
@@ -4,25 +4,65 @@ Papers, Presentations, and Posters
...
+  Gaffney, Peer-Timo Bremer, Tom Benson, Peter Robinson, and Luc
+  Peterson, Brian Spears, `"Parallelizing Training of Deep Generative
+  Models on Massive Scientific
+  Datasets" <https://doi.org/10.1109/CLUSTER.2019.8891012>`_, in `*2019
+  IEEE International Conference on Cluster Computing
+  (CLUSTER)* <https://clustercomp.org/2019/program/>`_, Albuquerque,
+  NM, USA, 2019, pp. 1-10, doi:
+  `10.1109/CLUSTER.2019.8891012<https://doi.org/10.1109/CLUSTER.2019.8891012>_`.::
+
+    @INPROCEEDINGS{8891012,
```
Comment (benson31) [inline docs/publications.rst:26]: I assume you're going to move this into it's own `rst` file? Also, the keys should be consistent.

Comment (bvanessen) [reply docs/publications.rst:26]: Yes, I am trying to get one to work.

File: docs/bib/Dryden2018.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Aluminum: An Asynchronous, GPU-Aware Communication Library Optimized for Large-Scale Training of Deep Neural Networks on HPC Systems
+======================================================================================================================================
+
+.. code-block:: bibtex
+
+    @INPROCEEDINGS{8638639,
+      author={N. {Dryden} and N. {Maruyama} and T. {Moon} and T. {Benson} and A. {Yoo} and M. {Snir} and B. {Van Essen}},
+      booktitle={2018 IEEE/ACM Machine Learning in HPC Environments (MLHPC)},
+      title={Aluminum: An Asynchronous, GPU-Aware Communication Library Optimized for Large-Scale Training of Deep Neural Networks on HPC Systems},
```
Comment (ndryden) [inline docs/bib/Dryden2018.rst:9]:
```suggestion
      title={Aluminum: An Asynchronous, {GPU}-Aware Communication Library Optimized for Large-Scale Training of Deep Neural Networks on {HPC} Systems},
```

File: docs/bib/Dryden2019spatial.rst [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Improving Strong-Scaling of CNN Training by Exploiting Finer-Grained Parallelism
+=================================================================================
+
+.. code-block:: bibtex
+
+    @INPROCEEDINGS{8820780,
+      author={N. {Dryden} and N. {Maruyama} and T. {Benson} and T. {Moon} and M. {Snir} and B. {Van Essen}},
+      booktitle={2019 IEEE International Parallel and Distributed Processing Symposium (IPDPS)},
+      title={Improving Strong-Scaling of CNN Training by Exploiting Finer-Grained Parallelism},
```
Comment (ndryden) [inline docs/bib/Dryden2019spatial.rst:9]:
```suggestion
      title={Improving Strong-Scaling of {CNN} Training by Exploiting Finer-Grained Parallelism},
```

**Review by ndryden — APPROVED** [review]
Looks good, only some minor suggestions.

**Review by benson31 — APPROVED** [review]

**Pull request merged**